### PR TITLE
Wheel vertical zoom events

### LINKED ISSUE #153: Support y-axis zoom with mouse scroll + shift
- note that excel/spreadsheet software uses this technique

- filters from before the zoom should stay

### PR DESCRIPTION
**Pull request by keller-mark**
This pull request fixes #153 . It allows vertical zooming with the mouse wheel/trackpad in horizontal-multivec tracks while the `Y` (or `y`) key is pressed.

There are still some minor issues but I think it may be good to include in the demo while we work on fixing them:
- I think the mouse position is not being accounted for correctly.
- I think there is a difference in the magnitude of zoom deltas between the trackpad and the mouse wheel. (Probably has to do with differences in the "deltaMode" in the wheel event https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent/deltaMode)
- zooming is not very useful without panning
- we should figure out a good way to notify the user that this feature is available since it is not obvious to press the `Y` key.

### CODE REVIEW

File: src/utils/options.js [hunk 1]
```diff
@@ -129,6 +130,23 @@ const baseSchema = {
                     "description": "Min and max values"
                 }
             }
+        },
+        "zoomInfo": {
+            "type": "object",
+            "properties": {
```
Comment (sehilyi) [inline src/utils/options.js:136]:
```suggestion
            "required": ["level", "top", "numRows"],
            "properties": {
```

Comment (sehilyi) [reply src/utils/options.js:136]: Just a minor thing, but can we expect that these keys will be provided every time?

File: src/TrackRowZoomOverlay.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * An overlay element to intercept wheel events in horizontal multivec tracks.
+ * @prop {number} trackX The x offset of the horizontal multivec track.
+ * @prop {number} trackY The y offset of the horizontal multivec track.
+ * @prop {number} trackWidth The width of the horizontal multivec track.
+ * @prop {number} trackHeight The height of the horizontal multivec track.
+ * @prop {function} onZoomRows The function to call on zoom events.
+ * @prop {boolean} isWheelListening Should the wheel events be intercepted right now?
+ * i.e. is the designated key pressed down?
+ */
+export default function TrackRowZoomOverlay(props) {
```
Comment (sehilyi) [inline src/TrackRowZoomOverlay.js:15]: Can you add this function to the documentation?

Comment (keller-mark) [reply src/TrackRowZoomOverlay.js:15]: Good catch, thanks!

**Pull request merged**